posts/tests.py

```diff
@@ -83,3 +95,51 @@ def test_resume_deletion_after_author_deletion(self, persist_resume):
     def test_resume_invalid_args(self, user, description, expected_error):
         with pytest.raises(expected_error):
             Resume(user=user, description=description)
+
+
+# -------------------------------------------- Poll tests --------------------------------------------
+
+
+@pytest.mark.django_db()
+class TestPoll:
+    # Check that the Poll values are the same as the Poll inputs.
+    def test_new_poll_input_same_as_output(self, new_poll):
+        assert new_poll.author.username == USERNAME
+        assert new_poll.description == DESCRIPTION
+
+    # Check if the Poll is saved in the database.
+    def test_persist_poll(self, persist_Poll):
+        assert persist_Poll in Poll.objects.all()
+
+    # Check if the Poll can be referred as post.
+    def test_inheritance_of_poll_from_post(self, persist_Poll):
+        assert isinstance(persist_Poll, Post)
```

Comment:
Same as Nati told me, you can use:
`assert Post.objects.filter(poll=persist_poll).exists()`
Instead of isinstance()

Reply:
Changed it 👍